lib/helpers/hearing_renderer.rb

```diff
@@ -0,0 +1,358 @@
+
+    formatted_text
+  end
+
+  def format_hearing_day_label(hearing_day)
+    formatted_text = "HearingDay #{hearing_day.id}"
+    formatted_text += " (#{get_ro_label(hearing_day.regional_office, hearing_day.request_type)}) "
+    formatted_text
+  end
+
+  def format_hearing_label(hearing)
+    hearing_type = print_nil(hearing.readable_request_type)
+    disposition = hearing.disposition.nil? ? "no disposition" : hearing.disposition
+    coordinator = hearing.bva_poc.nil? ? "no coordinator" : hearing.bva_poc
+
+    "#{hearing_type}, #{disposition}, #{coordinator}"
+  end
+
+  def shared_hearing_children(hearing)
+    children = []
+    children << "Notes: #{print_nil(hearing.notes)}" if show_pii
+    children << scheduled_for(hearing)
+    children << format_hearing_label(hearing)
+    children << format_hearing_day_label(hearing.hearing_day)
+    virtual_hearings = VirtualHearing.where(hearing_id: hearing.id, hearing_type: hearing.class.name)
+    children += virtual_hearings.map { |vh| structure(vh) }
+    children << {
+      "Email events": hearing.email_events.map do |ev|
+        "#{ev.recipient_role}, #{ev.email_type}, "\
+          "#{readable_date(ev.sent_at, false)}, #{ev.external_message_id}"
+      end
+    }
+    children << structure(hearing.hearing_task_association.hearing_task)
+  end
+
+  def legacy_hearing_children(hearing)
+    shared_hearing_children(hearing)
+  end
+
+  def legacy_hearing_details(hearing)
+    ["VACOLS ID: #{hearing.vacols_id}"]
+  end
+
+  def legacy_hearing_context(hearing)
+    hearing.appeal
+  end
+
+  def hearing_children(hearing)
+    shared_hearing_children(hearing)
+  end
+
+  def hearing_details(hearing)
+    ["UUID: #{hearing.uuid}"]
+  end
+
+  def hearing_context(hearing)
+    hearing.appeal
+  end
+
+  def hearing_task_children(hearing_task)
+    children = []
+    children << "instr: #{print_nil(hearing_task.instructions)}" if show_pii
+    children += hearing_task.children.map do |ct|
+      if ct.is_a?(ScheduleHearingTask) || ct.is_a?(AssignHearingDispositionTask)
+        "#{ct.class.name} #{ct.id} (#{ct.status}, ca: #{readable_date(ct.created_at)})"
+      end
+    end
+
+    children
+  end
+
+  def hearing_task_details(hearing_task)
+    ["#{hearing_task.status}, ca: #{readable_date(hearing_task.created_at)}"]
+  end
+
+  def hearing_task_context(hearing_task)
+    hearing_task.appeal
+  end
+
+  def email_sent(flag)
+    flag ? "email sent" : "email not sent"
+  end
+
+  def virtual_hearing_children(virtual_hearing)
+    children = []
+    recipients = []
+    recipients <<
+      "appellant - #{email_sent(virtual_hearing.appellant_email_sent)}" if virtual_hearing.appellant_email.present?
+    recipients << ", rep - " \
+      "#{email_sent(virtual_hearing.representative_email_sent)}" if virtual_hearing.representative_email.present?
+    recipients <<
+      ", judge - #{email_sent(virtual_hearing.judge_email_sent)}" if virtual_hearing.judge_email.present?
```

Comment:
suggesting codeclimate fixes
```suggestion
    recipients << if virtual_hearing.appellant_email.present?
        "appellant - #{email_sent(virtual_hearing.appellant_email_sent)}"
    end
    recipients << if virtual_hearing.representative_email.present? 
        ", rep - #{email_sent(virtual_hearing.representative_email_sent)}"
    end
    recipients << if virtual_hearing.judge_email.present?
        ", judge - #{email_sent(virtual_hearing.judge_email_sent)}"
    end
```